common/reportMetrics.py

```diff
@@ -444,9 +446,15 @@ def process_replay(file_path):
 
         else:  # metric file replay processing
             # handle different metric agents
-            if parameters['agentType'] == 'sar':
-                for command in ['sar', 'sar -n DEV', 'sar -p -d']:
-                    replay_sar(file_path, command)
+            if 'sar' in parameters['agentType']:
```

Comment:
now that you have already determine whether 'sar' in parameters['agentType'], why you put or condition below?

Reply:
-t sar will do each of sar-cpu, sar-mem, sar-network, and sar-storage